BodyweightFitness/AbstractViewController.swift

```diff
@@ -53,7 +53,7 @@ class AbstractViewController: UIViewController {
     func createBackgroundView(height: Int = 50) {
         let view = UIView()
         view.backgroundColor = UIColor.primary()
-        
+        self.contentView.backgroundColor = UIColor.white
```

Comment:
This can be deleted :-)

Reply:
Fixed, I'm going to send a new pull request